core/server/kustomization_test.go

```diff
@@ -132,11 +132,11 @@ func TestGetKustomization(t *testing.T) {
 	})
 }
 
-func newNamespace(ctx context.Context, k client.Client, g *GomegaWithT) *corev1.Namespace {
+func newNamespace(ctx context.Context, k client.Client, g *GomegaWithT) corev1.Namespace {
 	ns := &corev1.Namespace{}
 	ns.Name = "kube-test-" + rand.String(5)
 
 	g.Expect(k.Create(ctx, ns)).To(Succeed())
 
-	return ns
+	return *ns
```

Comment:
This can be
```go
func newNamespace(ctx context.Context, k client.Client, g *GomegaWithT) corev1.Namespace {
	ns := corev1.Namespace{}
	ns.Name = "kube-test-" + rand.String(5)

	g.Expect(k.Create(ctx, &ns)).To(Succeed())

	return ns
}
```
It's kind of the same, but looks less weird with `return *ns`.